core/src/main/java/org/kiwix/kiwixmobile/core/main/TabsAdapter.kt

```diff
@@ -65,6 +68,14 @@ class TabsAdapter internal constructor(
       .apply {
         id = R.id.tabsAdapterCloseImageView
         setImageDrawableCompat(R.drawable.ic_clear_white_24dp)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+          TooltipCompat.setTooltipText(this, "Close Tab")
```

Comment:
Please don't hardcode the string in the code, move this to the string file and use this from there.

Reply:
How to add, its showing this error : "close_tab" is translated here but not found in default locale

Reply:
Fixed it by Clean > Invalidate n Restart